vivisect/__init__.py

```diff
@@ -464,10 +464,10 @@ def delFuncAnalysisModule(self, modname):
         Example:
             vw.delFuncAnalysisModule('mypkg.mymod')
         '''
-        if not self.fmods.has_key(modname):
+        if modname not in self.fmods:
```

Comment:
more recently we've tended toward a paradigm like
```
x = self.fmods.pop(modname, None)
if x is None:
    raise Stuff
self.fmodlist.remove(modname)
```
it saves a hash calc and dict lookup (and in this case an additional condition check).  lets start that trend in vivi town